### PR TITLE
adding flexibility to select which chambers are used

### PR DESCRIPTION
**Pull request by jsturdy**
Just for review of what has been done, still needs to be tested

### CODE REVIEW

**Comment by jsturdy** [comment]
references #70

File: anaAllChambers.py [hunk 1]
```diff
@@ -1,41 +1,98 @@
...
+        print "No file to analyze. %s does not exist"%(filename)
+        return
+
+    cmd1 = ["python","%s/vfatqc-python-scripts/macros/anaUltraScurve.py"%(os.getenv("BUILD_HOME"))]
+    cmd1.append("--infilename=%s"%(filename))
+    cmd1.append("--fit")
+    cmd1.append("--type=%s"%(cType))
+
+    cmd2 = ["cp","%s/%s/scurve/%s/SCurveData/Summary.png"%(dataPath,cName,scandate),
+            "~/move/SCurveSummary_%s_ztrip%2.2f.png"%(cName,ztrim)]
```
Comment (cbravo135) [inline anaAllChambers.py]: typo here: ztrip->ztrim

File: anaAllChambers.py [hunk 2]
```diff
@@ -1,41 +1,98 @@
...
+    import subprocess
+    import itertools
+    from multiprocessing import Pool, freeze_support
+    from chamberInfo import chamber_config,GEBtype
+    from gempython.utils.wrappers import envCheck
+    import glob
+
+    from qcoptions import parser
+
+    parser.add_option("--scandate", type="string", dest="scandate", default="current",
```
Comment (cbravo135) [inline anaAllChambers.py:48]: good idea, I was already thinking about this same thing

**Review by cbravo135 — COMMENTED** [review]
So the plan is to comment out lines in the def'n of chamber config? Just one typo you should fix, then I will approve.

**Comment by jsturdy** [comment]
one other question, in ```run_scans.py``` there was this line:
```    setupCmds.append( ["ln","-s",startTime,dirPath+"current"] )```
and it wasn't clear to me that it shouldn't be:
```    setupCmds.append( ["ln","-s",dirPath+startTime,dirPath+"current"] )```
clearly it's been working, so I know it works, so probably not a bug

**Comment by cbravo135** [comment]
it is safer to use a relative path for the link than the full path, in the case you want to copy the whole directory somewhere.

**Review by cbravo135 — APPROVED** [review]

**Pull request merged**